app/views/dashboard/shared/_thumbnail_form_fields.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="row p-2">
+  <%= fields_for :thumbnail_form, thumbnail_form do |field| %>
+    <div class="col-sm-4 d-flex justify-content-center">
+      <div>
+        <%= field.radio_button :thumbnail_selection, ThumbnailSelections::DEFAULT_ICON.to_sym %>
+        <%= field.label :thumbnail_selection,
+                        'Use Default Icon',
+                        value: ThumbnailSelections::DEFAULT_ICON.to_sym %>
+      </div>
+    </div>
+    <div class="col-sm-4">
+      <div class="row d-flex justify-content-center">
+        <% if thumbnail_form.resource.thumbnail_upload.present? %>
```

Comment:
There's a lot of logic in this file like this control flow.  I tried refactoring this into a view component, but it just made things more complicated and less testable imo.  Perhaps a presenter class would be better, or maybe it's fine as it is.